cmd/update.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const githubSlug = "screwdriver-cd/sd-local"
+
+var (
+	currentVersion = version
+	updateFlag     = false
+)
+
+func canUpdate() (*selfupdate.Release, error) {
+
+	if currentVersion == "dev" {
+		return &selfupdate.Release{}, errors.New("This is a development version and cannot be updated")
+	}
+
+	latest, found, err := selfupdate.DetectLatest(githubSlug)
+	if err != nil {
+		return &selfupdate.Release{}, err
+	}
+	if !found {
+		return &selfupdate.Release{}, errors.New("Repositry Not Found")
+	}
+	v := semver.MustParse(currentVersion)
+
+	if latest.Version.LTE(v) {
+		return &selfupdate.Release{}, errors.New("Current version is latest")
+	}
+
+	return latest, nil
+}
+
+func checkUserInput(input string) error {
+	if input == "y" || input == "Y" {
+		return nil
+	}
+
+	if input == "n" || input == "N" || input == "" {
+		return errors.New("Aborted")
```

Comment:
I think this case is not an error.
It's better to use `logrus` to output a message like "aborted the update".